subprojects/docs/src/docs/userguide/userguide.xml

```diff
@@ -106,6 +107,7 @@
         <xi:include href='../../../build/asciidoc/userguide/tutorialGroovyProjects.xml'/>
         <xi:include href='../../../build/asciidoc/userguide/groovyPlugin.xml'/>
         <xi:include href='../../../build/asciidoc/userguide/scalaPlugin.xml'/>
+        <xi:include href='../../../build/asciidoc/userguide/basePlugin.xml'/>
```

Comment:
We may want to list the base plugin first as it is the basis for the Java, Groovy, Scala plugin.

Reply:
Ummm...it's kind of irrelevant as the order in this file is not reflected in the generated user manual. However, the file isn't processed unless it's added to this DocBook index.

Reply:
Isn't this the index of the single page HTML user guide? Are you referring to the one without the left navigation bar generated with `:docs:userguideSingleHtml`?

Reply:
The layout of the manual at /userguide/userguide.html is specified by https://github.com/gradle/gradle/blob/master/subprojects/docs/src/docs/stylesheets/userGuideHtml.xsl. You're right, though, that the TOC for the single page manual is derived from _userguide.xml_. Thanks for the pointer and I've already updated the order. It's still in the Java section, because I'm not sure where else is suitable for it. Unless we add a Plugin Reference part to the manual.